client/src/pages/MatchWinScreen.tsx

```diff
@@ -0,0 +1,65 @@
+import plainTrophy from "../assets/PlainTrophy.svg";
+import PlayerName1 from "../components/PlayerNameMobile1";
+import PlayerName2 from "../components/PlayerNameMobile2";
+import SpectatorCount from "../components/SpectatorCount";
+import StarsInfoPlayer from "../components/StarsInfoPlayer";
+import StarsInfoOpponent from "../components/StarsInfoOpponent";
+
+type MatchWinScreenProps = {
+  playerScore: number;
+  opponentScore: number;
+  playerName: string;
+  opponentName: string;
+  spectatorCount: number;
+};
+
+const MatchWinScreen = ({
```

Comment:
Instead of creating two different screens entirely, can we just use one component and create a variable called `isWin`?

Then for changing the text do something like this:

`{isWin ? "YOU WON" : "YOU LOST"}`

This will change the text based on what prop you pass.

You can do the same thing for the image.